code/game/objects/machinery/sentries.dm

```diff
@@ -838,12 +838,19 @@
 	return
 
 /obj/machinery/marine_turret/proc/load_into_chamber()
-	if(in_chamber) return 1 //Already set!
-	if(!CHECK_BITFIELD(turret_flags, TURRET_ON) || !cell || rounds == 0 || machine_stat == 1) return 0
+	if(in_chamber)
+		return TRUE //Already set!
+	if(!CHECK_BITFIELD(turret_flags, TURRET_ON) || !cell || rounds == 0 || machine_stat & (NOPOWER|BROKEN))
```

Comment:
```suggestion
	if(!CHECK_BITFIELD(turret_flags, TURRET_ON) || !cell || !rounds || machine_stat & (NOPOWER|BROKEN))
```

Reply:
I prefer this format because `!var` implies the variable is a boolean.